### ISSUE TITLE
Win32EventLogAppender.dll build Fails in local machine with WindRes

### ISSUE THREAD

**Issue by claudiow** [dblock/log4jna#29]
When running the build with windres I get this:
```
[INFO] --- maven-antrun-plugin:1.8:run (add-dll) @ win32dll ---
[INFO] Executing tasks

main:

clean:
   [delete] Deleting directory C:\Users\claudiow\git\log4jna\win32dll\target

ver:
     [copy] Copying 1 file to C:\Users\claudiow\git\log4jna\win32dll\src

windres:
    [mkdir] Created dir: C:\Users\claudiow\git\log4jna\win32dll\target\obj
     [exec] windres: can't open file `MSG00001.bin': No such file or directory
     [exec] Result: 1

compile:

windres-echo:
     [echo] windres could not be found or failed, Win32EventLogAppender.dll build skipped.

build:
[INFO] Executed tasks
```
The **MSG00001.bin** is not found, and that is OK as it is deleted by the `clean `task.

I soupouse thet it should be generated by the `windres`task

**Comment by dblock** [comment]
This is fixed in #28, no?

**Comment by claudiow** [comment]
Nope, I'm just fixing it. The cause is that the it was missing a call to `windmc` teh build was calling 'windres' without compiling the message first.

I'm creating a 2 profile run in the `pom.xml` that will activate VC compile by default on windows machines and MinGW in Linux or UN*X machines.

I'm enforcing VC in windows to avoid changes between developer machines an CI environment.
